# Quick and dirty workaround for docs build failure, until upstream comes up
# with an actual fix for the broken function in docs/conf.py.
#
# See https://github.com/cherrypy/cherrypy/issues/1970
--- cherrypy3-18.7.0.orig/docs/conf.py
+++ cherrypy3-18.7.0/docs/conf.py
@@ -38,7 +38,11 @@ def get_supported_pythons(classifiers):
     vers = map(lambda c: tuple(c.split('.')), vers)
     vers = sorted(vers)
     del vers[1:-1]
-    return vers
+    if len(vers) > 0:
+        return vers
+    else:
+        current_version = (str(sys.version_info.major), str(sys.version_info.minor))
+        return [current_version, current_version]
 
 
 custom_sphinx_theme = try_import('alabaster')
